docs/src/pages/components/data-grid/events/CatalogOfEventsNoSnap.js

```diff
@@ -10,33 +23,117 @@ const useStyles = makeStyles({
   },
 });
 
+function escapeHTML(value) {
+  return value.replace(/</g, '&lt;').replace(/>/g, '&gt;');
+}
+
+const EventRow = ({ event }) => {
+  const [open, setOpen] = React.useState(false);
+
+  const example = React.useMemo(() => {
+    const args = ['details // GridCallbackDetails'];
```

Comment:
When multiline, all arguments have a trailing comma.

```suggestion
    const args = ['details, // GridCallbackDetails'];
```